### PR TITLE
Develop audio icon

### LINKED ISSUE #1845: Replace Language code with icon in AudioBar
Currently Audio Bar in exploration has a language code, as shown below:
Screenshot_1600232780

We should replace the language code to [icon](https://drive.google.com/file/d/1jPRimZBxcPHeEHwlmTUcLrDcwZyVt0gl/view?usp=sharing) as shown below:
Screenshot 2020-09-16 at 10 37 42 AM

Reference: https://xd.adobe.com/view/e8aa4198-3940-47f9-514a-f41cc54457f6-9e9b/screen/d74f0571-471f-4ea6-a3a5-32e135253c56/

The tests in AudioFragmentTest need to be updated (see the TODOs there).

### PR DESCRIPTION
**Pull request by MaskedCarrot**
## Explanation

Fixes #1845 
Replaced audio language text with an icon

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [ ] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [ ] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by aggarwalpulkit596** [comment]
The test are failing could you check which test are failing also please don't create seperate PR dor changes.

**Comment by MaskedCarrot** [comment]
I cannot figure out why the build is failing can you please help

**Comment by rt4914** [comment]
Hello @MaskedCarrot I just want to confirm if you have followed the instructions mentioned on https://github.com/oppia/oppia-android/wiki#onboarding-instructions . Please follow the instructions and especially sign the CLA and contributors survey form, once that is done, one of the mentors will contact you. Thanks.

@aggarwalpulkit596 Thanks for helping out @MaskedCarrot . Please hold this PR until you get confirmation that the student has signed the CLA and filled the contributor survey form. 

Adding @BenHenning if I am not available.

**Comment by aggarwalpulkit596** [comment]
Could you please first fill up the form that i suggested you on the issue thread ? Let me know when you're done with that

**Comment by MaskedCarrot** [comment]
I have already filled the form you asked me

**Comment by MaskedCarrot** [comment]
I have filled both the forms

**Comment by rt4914** [comment]
> I have filled both the forms

Thanks @MaskedCarrot . I have sent you one welcome email, have a look.

@anandwana001 @aggarwalpulkit596 Can you please help him in his first PR? Thanks.

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -5,6 +5,22 @@
       <option name="NAMES_COUNT_TO_USE_IMPORT_ON_DEMAND" value="999" />
     </JavaCodeStyleSettings>
     <JetCodeStyleSettings>
+      <option name="PACKAGES_TO_USE_STAR_IMPORTS">
+        <value>
```
Comment (prayutsu) [inline .idea/codeStyles/Project.xml:9]: @MaskedCarrot I guess you should not add any changes to this file.

Comment (aggarwalpulkit596) [reply .idea/codeStyles/Project.xml:9]: Yeah that's correct

**Comment by aggarwalpulkit596** [comment]
@MaskedCarrot  two of the tests are failing could you please fix those also revert the project.xml file

**Comment by MaskedCarrot** [comment]
can you please help with fixing the code in codeStyles/Project.xml

**Comment by aggarwalpulkit596** [comment]
@MaskedCarrot you have to revert that file that's it nothing to be fixed there

**Comment by MaskedCarrot** [comment]
What i ment was these lines get added automatically when I build the project

**Comment by aggarwalpulkit596** [comment]
But you can avoid to committ this file ?

**Comment by MaskedCarrot** [comment]
okay

**Pull request closed without merging**